google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/flexible/AppEngineFlexibleDeploymentEditor.java

```diff
@@ -158,6 +158,9 @@ protected void textChanged(DocumentEvent event) {
     appYamlOverrideCheckBox.addActionListener(event -> {
       boolean isAppYamlOverrideSelected = ((JCheckBox) event.getSource()).isSelected();
       appYamlTextField.setVisible(isAppYamlOverrideSelected);
+
+      setModuleComboBoxAndButtonEnabled(
```

Comment:
yea I guess you're right. It felt a little weird with it being long. But I have no problem with setModuleControlsEnabled/State